### PR TITLE
refactor(test): integration tests for applications

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

refactor: application integration tests

## Testing

Integration tests passed.

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/integration-tests/tests/applications.test.ts [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+const createApplication = (name: string, type: ApplicationType) => {
+  return authedAdminApi
+    .post('applications', {
+      json: {
+        name,
+        type,
+      },
+    })
+    .json<Application>();
 };
```
Comment (IceHe) [inline packages/integration-tests/tests/applications.test.ts:15]: simplify
```suggestion
const createApplication = (name: string, type: ApplicationType) =>
  authedAdminApi
    .post('applications', {
      json: {
        name,
        type,
      },
    })
    .json<Application>();
};
```

Comment (IceHe) [inline packages/integration-tests/tests/applications.test.ts:15]: simplify
```suggestion
const createApplication = (name: string, type: ApplicationType) =>
  authedAdminApi
    .post('applications', {
      json: {
        name,
        type,
      },
    })
    .json<Application>();
```

File: packages/integration-tests/tests/applications.test.ts [hunk 2]
```diff
@@ -19,35 +24,29 @@ describe('admin console application', () => {
...
+    const applicationType = ApplicationType.SPA;
 
-    expect(application.name).toBe(testApplication.name);
-    expect(application.type).toBe(testApplication.type);
+    const application = await createApplication(applicationName, applicationType);
 
-    // eslint-disable-next-line @silverhand/fp/no-mutation
-    testApplication.id = application.id;
+    expect(application.name).toBe(applicationName);
+    expect(application.type).toBe(applicationType);
```
Comment (IceHe) [inline packages/integration-tests/tests/applications.test.ts:33]: nit: since we have used `toEqual` in all other tests, better align `toEqual`?

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: packages/integration-tests/tests/applications.test.ts (hunk 2 above)
Comment (IceHe) [inline packages/integration-tests/tests/applications.test.ts:33]:
```suggestion
    expect(application.name).toEqual(applicationName);
    expect(application.type).toEqual(applicationType);
```

Comment (xiaoyijun) [reply packages/integration-tests/tests/applications.test.ts:33]: `toEqual` is used when you want to check that two objects have the same value, so we should use `toBe` here.

**Pull request merged**